### PR TITLE
Feature/sidekiq implementation

### PR DESCRIPTION
**Pull request by alexesba**
*Add Sidekiq and Sidekiq scheduler configuration files to allow sync the users from bamboo every day at 6am.

### CODE REVIEW

**Comment by zalapeach** [comment]
LGTM, but if you could add tests for this would be awesome!

File: config/routes.rb [hunk 1]
```diff
@@ -33,8 +33,10 @@
   end
 
   scope 'admin' do
+    require 'sidekiq/web'
     mount BadgesEngine::Engine , at: '/'
     mount PresentationsEngine::Engine, at: '/'
+    mount Sidekiq::Web => '/sidekiq'
```
Comment (mumoc) [inline config/routes.rb]: do you need the sidekiq web on prod? you could check for env and load it only for dev ...

File: db/schema.rb [hunk 2]
```diff
@@ -197,14 +197,4 @@
   add_index "votes", ["user_id"], name: "index_votes_on_user_id", using: :btree
   add_index "votes", ["votation_id"], name: "index_votes_on_votation_id", using: :btree
 
-  create_table "workshops_engine_workshops", force: true do |t|
```
Comment (mumoc) [inline db/schema.rb]: Does these changes affect your implementation?

File: config/scheduler.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+sync_users_from_bambo_api:
+  cron: "0 6 * * *"
+  class: SyncUsersJob
+  queue: users
+  description: "This job will sync the users every 30 seconds after 120 seconds after start"
```
Comment (mumoc) [inline config/scheduler.yml]: Is the description correct? I mean, evey 30 seconds after 120 seconds? shouldn't it be every day at 6 am???

Comment (alexesba) [inline config/scheduler.yml]: jeje, sorry for that.

File: db/schema.rb (hunk 2 above)
Comment (alexesba) [inline db/schema.rb]: Those changes appeared after run the migrations on my local. Somehow the schema.rb wasn't updated.

File: config/routes.rb (hunk 1 above)
Comment (alexesba) [inline config/routes.rb]: I don't know, @vicmaster do we need the admin section for the jobs?

**Comment by vicmaster** [comment]
LGTM

File: db/schema.rb (hunk 2 above)
Comment (mumoc) [inline db/schema.rb]: got it, remember to send the schema always (whoever to modify this repo)

File: config/routes.rb [hunk 4]
```diff
@@ -35,6 +35,7 @@
   scope 'admin' do
     mount BadgesEngine::Engine , at: '/'
     mount PresentationsEngine::Engine, at: '/'
+    mount Sidekiq::Web => '/sidekiq'
```
Comment (mumoc) [inline config/routes.rb]: @vicmaster you missed a question here .... before marking it as LGTM xD

Comment (vicmaster) [inline config/routes.rb]: We need it to check that in the admin in production env

**Pull request merged**